ring_api/extra/servers/curl_requests.txt

```diff
@@ -1,66 +0,0 @@
-GET: ?param=value
-POST: json
-
-/account/
-GET: curl http://localhost:8080/account/\?type\=SIP
-POST: curl -d '{"details":{"foo":"bar"}}' http://localhost:8080/account/
```

Comment:
@Bl4ckb0ne I think it's a good idea to keep it for some folks that want to know how to craft curl requests. If it's incomplete, it is okay, just add a comment stating it. If you moved it, don't forget to recommit it!